app/static/style.css

```diff
@@ -331,6 +336,19 @@ outline: 0;
     margin-top: -5px;
 }
 
+.profile-user-settings .btn {
+    margin-left: 10px;
+    border-radius: 5px;
+    font-size: 12px;
+    font-weight: bold;
+}
```

Comment:
Add `margin-top: -9px;`

Reply:
- added top margin for the follow button